src/seedsigner/views/seed_views.py

```diff
@@ -1484,9 +1486,6 @@ def run(self):
         for seed in seeds:
             button_str = seed.get_fingerprint(self.settings.get_value(SettingsConstants.SETTING__NETWORK))
             
-            if seed.passphrase is not None:
-                # TODO: Include lock icon on right side of button
-                pass
```

Comment:
Decided that it was not desired to have a lock icon as described in this comment (see [here](https://github.com/SeedSigner/seedsigner/pull/286#issuecomment-1363351268) for discussion). Removing comment and if statment.

Reply:
TY @EverydayBitcoiner, this code cleanup has been included in pr #324